### DISCUSSION TITLE
dont synchronize my wallet

### DISCUSSION THREAD

**Discussion by tuniflor68** [Chia-Network/chia-blockchain#18880] [Support]
Hi, 
I used translation software to translate this response and apologize for any inaccuracies. 
Gist of the appeal:
I made a transfer of coins from the exchange to my CHIA wallet on 05.08.2024.
The coins did not arrive to my wallet. The transaction is confirmed and is on the correct xch-address
Wallet is not fully synchronized 
Help me how to synchronize my wallet

**Comment by BrandtH22** [comment]
Hey @tuniflor68 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia). 

A few questions:
- what version of chia are you running? (if older than 2.0 please update and attempt to resync)
- are there any errors in the logs? (they can be found in `~/.chia.mainnet/log/debug.log`)
- does the wallet fully sync? (i.e. is the wallet sync icon in the top right green?)

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
